instrumentation/apache-httpclient/apache-httpclient-5.2/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v5_2/internal/Experimental.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachehttpclient.v5_2.internal;
+
+import io.opentelemetry.instrumentation.apachehttpclient.v5_2.ApacheHttpClientTelemetryBuilder;
+import java.util.function.BiConsumer;
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and experimental. Its APIs are unstable and can change at any time. Its
+ * APIs (or a version of them) may be promoted to the public stable API in the future, but no
+ * guarantees are made.
+ */
+public final class Experimental {
+
+  @Nullable
+  private static volatile BiConsumer<ApacheHttpClientTelemetryBuilder, Boolean>
+      setEmitExperimentalTelemetry;
+
+  public static void setEmitExperimentalTelemetry(
+      ApacheHttpClientTelemetryBuilder builder, boolean emitExperimentalTelemetry) {
+    if (setEmitExperimentalTelemetry != null) {
+      setEmitExperimentalTelemetry.accept(builder, emitExperimentalTelemetry);
+    }
+  }
+
+  public static void setSetEmitExperimentalTelemetry(
```

Comment:
typo in method name

Reply:
it's "setting" the field called "setEmitExperimentalTelemetry", do you have a preference on a different naming, I'm happy to change?

Reply:
I think this method can be deleted.

Reply:
ah ok I see now what this method actually does

Reply:
I don't follow, this is the method that's called from the Builder static initializer